import_email_events.py

```diff
@@ -0,0 +1,17 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL,
+    locale VARCHAR(40),
+    domain VARCHAR(40),
+    template VARCHAR(64),
+    flow_id VARCHAR(64)
```

Comment:
`flow_id` should be promoted to `DISTKEY` here, and it might be worth reading the redshift docs about `DISTKEY` and `SORTKEY` if you haven't already. They're pretty integral to query performance. (also, note that the `timestamp` column is automatically promoted to `SORTYKEY` inside `import_events.py`)

We probably also want to compress every column. @rfk ran some magic incantation on our other tables that said `ENCODE lzo` was the best setting elsewhere in this regard, so presumably it will be best here too.

You can see examples for both of the above in `import_flow_events.py`.